docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,404 @@
+
+```bash
+aztec-wallet create-account -a alice --payment method=fee_juice,feePayer=test0
+```
+
+:::warning
+Specifying the `feePayer` is only an option for commands that deploy an account.
+:::
+
+The equivalent using aztec.js - Create a fee juice payment method with a test account as sender, then deploy an account:
+
+```javascript
+import { FeeJuicePaymentMethod } from "@aztec/aztec.js";
+  //... building upon the previous section
+  const useFeeJuice = new FeeJuicePaymentMethod(testWallets[0].getAddress())
+  await schnorrAccount.deploy({ fee: { paymentMethod: useFeeJuice }}).wait()
+```
+
+:::note Payment: Sponsored Fee Paying Contract
+Options for explicitly stating fee_juice from the sender which is the default payment method:
+- CLI: `--payment method=fee_juice` (default)
+- .js: `{ fee: { paymentMethod: new FeeJuicePaymentMethod(<fee payer address>) }`
+:::
+
+### Bridging Fee Juice
+
+The sandbox allows free-minting on it's L1 to be bridged and claimed on its Aztec node.
+For testnet you'll first need an L1 account with the fee asset to send the fees to the Aztec bridge.
+
+We'll register a new account `accBFJ` and bridge fee-juice to it.
+
+```bash
+aztec-wallet create-account -a accBFJ --register-only
+aztec-wallet bridge-fee-juice 1000000000000000000 accBFJ --mint --no-wait
+```
+
+You'll have to wait for two blocks to pass for bridged fee juice to be ready on Aztec. For the sandbox you can do this by putting through two arbitrary transactions. Eg:
+```bash
+aztec-wallet deploy counter_contract@Counter --init initialize --args 0 accounts:test0 --from test0 -a counter
+aztec-wallet send increment -ca counter --args accounts:test0 accounts:test0 --from test0
+```
+
+Now the funded account can deploy itself with the bridged fees, claiming the bridged fee juice and deploying the contract in one transaction:
+
+```bash
+aztec-wallet deploy-account --from accBFJ --payment method=fee_juice,claim
+```
+
+
+The equivalent using aztec.js - bridge fee juice, (pass two txs), create and use payment method:
+
+(See [here](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/cli-wallet/src/cmds/bridge_fee_juice.ts#L32) to initialise a fee juice portal manager)
+
+```javascript
+import { FeeJuicePaymentMethod, PrivateFeePaymentMethod, PublicFeePaymentMethod } from "@aztec/aztec.js";
+  const newWallet = await schnorrAccount.getWallet()
+  const feeJuiceReceipient = schnorrAccount.getAddress()
+
+  const claim = await feeJuicePortalManager.bridgeTokensPublic(feeJuiceReceipient, 1000000000000000000n, true);
+  // ...(wait or perform two txs)
+  const claimAndPay = new FeeJuicePaymentMethodWithClaim(newWallet, claim)
+  await schnorrAccount.deploy({ fee: { paymentMethod: claimAndPay }}).wait()
+
+```
+
+:::note  Payment: Fee Juice with claim from bridge
+Options for claim+pay with bridged funds that can be used in multiple commands:
+- CLI: `--from <sender address> --payment method=fee_juice,claim`
```

Comment:
I think it would be good to take the code snippets out of the :::note block here too. It would look better outside and in the context of a full command / JS call

Reply:
Presented better as above. Note blocks are for quick scrolling reference. Can change if the tweak doesn't make sense still